Update how-to guide for offline installation (#5440)
vmware-tanzu/kubeapps: site/content/docs/latest/howto/offline-installation.md

- upgrade, manage and delete the applications that are deployed in your Kubernetes cluster
- expose an API to manage your package repositories and your applications

This guide explains in detail how to **install Kubeapps in an air-gapped environment**.

## Pre-requisites

To be able to able to install Kubeapps without an Internet connection, it's necessary to:

- Kubernetes cluster (air-gapped).
- Pre-download the Kubeapps Helm chart.
- Mirror Kubeapps images so they are accessible within the cluster.
- [Optional] Have one or more offline Package Repositories.

> **Note**: Internet connection is necessary at this point to download charts and images for an offline installation

## Step 1. Download the Kubeapps chart

First, download the tarball containing the Kubeapps chart from the publicly available repository maintained by Bitnami.

```bash
helm pull --untar https://charts.bitnami.com/bitnami/kubeapps-x.y.z.tgz
helm dep update ./kubeapps
```

> Notice `x.y.z` must be replaced by the latest version of the Kubeapps chart available at the [Bitnami Chart Repository](https://github.com/bitnami/charts/blob/master/bitnami/kubeapps/Chart.yaml#L3)

## Step 2. Mirror Kubeapps images

> To be able to install Kubeapps, it's necessary to either have a copy of all the images that Kubeapps requires in each node of the cluster or push these images to an internal Docker registry that Kubernetes can access. You can obtain the list of images by checking the `values.yaml` of the chart. For example:

Reviewer: ```suggestion
To be able to install Kubeapps, it's necessary to either have a copy of all the images that Kubeapps requires in each node of the cluster or push these images to an internal Docker registry that Kubernetes can access. You can obtain the list of images by checking the [`values.yaml` file](https://github.com/bitnami/charts/blob/master/bitnami/kubeapps/values.yaml) of the chart. For example:
```

```yaml
registry: docker.io
repository: bitnami/nginx
tag: 1.19.2-debian-10-r32
```

> The list of images to download includes (could change in time):
>
> - `bitnami/nginx`
> - `bitnami/kubeapps-dashboard`
> - `bitnami/kubeapps-apprepository-controller`
> - `bitnami/kubeapps-asset-syncer`
> - `bitnami/oauth2-proxy`
> - `bitnami/kubeapps-pinniped-proxy`
> - `bitnami/kubeapps-apis`
> - `bitnami/postgresql`

> For simplicity, in this guide, use a single-node cluster created with [Kubernetes in Docker (`kind`)](https://github.com/kubernetes-sigs/kind) with a namespace called "kubeapps". In this environment, the images have to be preloaded: - first, pull the images (`docker pull`), - next load them into the cluster (`kind load docker-image`).

Reviewer: In the rest of the document you're using `pre-xxxxx`, so changing it for consistency:

```suggestion
For simplicity, in this guide, use a single-node cluster created with [Kubernetes in Docker (`kind`)](https://github.com/kubernetes-sigs/kind), using a namespace called "kubeapps". In this environment, the images have to be pre-loaded: - first, pull the images (`docker pull`), - next load them into the cluster (`kind load docker-image`).
```

```bash
docker pull bitnami/nginx:1.19.2-debian-10-r32
kind load docker-image bitnami/nginx:1.19.2-debian-10-r32
```

> > **Note**: tags must be updated according to the images version in the `values.yaml` file

Reviewer: ```suggestion
> **Note**: tags must be updated according to the version of the images in the [`values.yaml` file](https://github.com/bitnami/charts/blob/master/bitnami/kubeapps/values.yaml)
```
